hoodie-hive/src/main/java/com/uber/hoodie/hive/HiveSyncTool.java

```diff
@@ -56,10 +60,16 @@
   private final HoodieHiveClient hoodieHiveClient;
   public static final String SUFFIX_REALTIME_TABLE = "_rt";
   private final HiveSyncConfig cfg;
+  private final boolean isOrcFile;
 
   public HiveSyncTool(HiveSyncConfig cfg, HiveConf configuration, FileSystem fs) {
+    this(cfg, configuration, fs, false);
+  }
+
+  public HiveSyncTool(HiveSyncConfig cfg, HiveConf configuration, FileSystem fs, boolean isOrcFile) {
```

Comment:
As pointed above, please create a HoodieFileFormat class with enum, pass that here and perform operations based on the enum..